ci/infra/testrunner/skuba/skuba.py

```diff
@@ -223,3 +229,18 @@ def _run_skuba(self, cmd, cwd=None, verbosity=None, ignore_errors=False):
             cwd=cwd,
             ignore_errors=ignore_errors,
         )
+
+    def _run_os_cmd(self, cmd, cwd=None, ignore_errors=False):
```

Comment:
I only see you are using this `_run_os_cmd` once. Any reason for not calling `runshlelcommand` directly?